### ISSUE TITLE
Test: Checkout Hawaiian configurations with no MP and MATS reporting requirements

### ISSUE THREAD

**Issue by ibarra-michelle** [US-EPA-CAMD/PI-1#6414]
RT # 74716 and 72607.

We need to verify if Hawaiian facilities, such as ORIS 765 and 766, are able to checkout their facilities and make MATS submissions. As of 09/30/24, Scott shared that issue is blocked by a bug.

FYI - From CBSIstagei, I am able to add ORIS 765 and 765 to my test account and view both facilities in the ECMPS 2.0 test env. However, I am not able to "checkout" a configuration to submit a MATS PDF. 

![Image](https://github.com/user-attachments/assets/d718a377-ea76-4dbe-872d-d4bfc8efee56)

**Comment by annalbrecht** [comment]
Blocked by #5995 and #6425.

**Comment by esaber76** [comment]
@ibarra-michelle: Once in Beta, they can use Configuration Management to create a new MP. Once created, they can check out the unit in QA and attempt to import MATS Data (**Note: my understanding is that this will be removed from this screen and moved to its own screen similar to ECMPS 1.0**). As it stands right now though, **the Import button would be disabled**. It appears that this is because there is no data available in the current Test Type and Test Number dropdowns based on how this was originally designed. If I submit a QA test for this location, I can import and submit a MATS Data file. This entire module will be redesigned to more closely align with ECMPS 1.0 so this part shouldn't be an issue in the future. 

I honestly don't know much about these units, but if they are required by the MATS rule to submit PM data (not sure if they are also supposed to be submitting for other MATS pollutants - others at EPA should know more about these units), they could build and submit a new MP that includes at a minimum a PM method, system, component, formula, etc. The check engine will tell them what it wants. Error suppressions may be needed similar to other MATS-only units. ECMPS 1.0 2024 Q1 check updates also need to get migrated to ECMPS 2.0 still.

The initial release of ECMPS 2.0 will obviously not contain the needed updates to submit PM QA tests and compliance reports via the QA module, or hourly PM emissions data via the Emissions module, so my understanding is they will continue to use the MATS Data Submission module for xml, json, and/or PDF submissions until such time that ECMPS 2.0 is updated. 

They could also start early depending on what they are supposed to be doing based on the MATS rule and build an MP in ECMPS 1.0, which will eventually come over in a data refresh with the MP already created. These screens are all dependent on the existence of a monitor_plan record in the database.

I did add and submit a bare bone MP that can currently pass the checks for ORIS 764, 16 in test as an example. There are still some Configuration Management changes going on that can affect this, particularly the setting of begin_rpt_period_id based on the earliest method. 

We can talk through this during standup if you would like. I can also demo the situation if needed.

**Comment by ibarra-michelle** [comment]
FYA @bryanramirez1 
@esaber76 thank you for the updates and suggestion on possibly starting early. I am bring this topic to @bryanramirez1 awareness so he can become more familiar with this use case and solution in development.

> They could also start early depending on what they are supposed to be doing based on the MATS rule and build an MP in ECMPS 1.0, which will eventually come over in a data refresh with the MP already created. These screens are all dependent on the existence of a monitor_plan record in the database.

**Comment by bryanramirez1** [comment]
FYA @esaber76 @ibarra-michelle after EPA discussion with ERG, decision is to notify facility at later date to create the MP in the workspace via config management initially and to not evaluate. Note: recheck/retest this procedure/workflow once the MATS module is completed.

To decide: create a new ticket to test this post completion of the MATS module OR move this ticket to a later sprint to retest.

**Comment by ibarra-michelle** [comment]
At DSU, the team determined that the functionality for this ticket has been addressed and a new ticket (#6561) will be created to test this post completion of the MATS functionality.

FYA @bryanramirez1

**Comment by ibarra-michelle** [comment]
@esaber76 @mark-hayward-erg @annalbrecht @spetros-do 

I am moving this ticket from QA Done to Ready for Deployment as we determined testing needs to happen POST MATS implementation in ECMPS 2.0.

FYA - @bryanramirez1 @maheese @j-tafoya @chrisgitsit

**Comment by esaber76** [comment]
I think we can close this one and mark as done. #6561 will handle any follow-on tasks.
